app/js/lib/reducers/form_content.js

```diff
@@ -5,9 +5,31 @@ var objectAssign = require( 'object-assign' ),
 	initialState = {
 		amount: 0,
 		isCustomAmount: false,
-		paymentType: 'BEZ'
+		paymentType: 'BEZ',
+		recurringPayment: 0, // 0 = one-off, 1 = recurring Payment
+		paymentPeriod: 0 // 0, 1, 3, 6 or 12
```

Comment:
Is it really easier to have two fields here? Looks like `recurringPayment` is made redundant by `paymentPeriod` (assuming 0 for the former implies 0 for the later).  If so, then an `isRecurring` method removes the need for the first field no?